packages/eslint-plugin/rules/jsdoc-requires-since.test.js

```diff
@@ -235,19 +235,5 @@ ruleTester.run( 'jsdoc-requires-since', rule, {
 				},
 			],
 		},
-		{
```

Comment:
Having updated `eslint-plugin-jsdoc`, the `jsdoc-requires-since` rule no longer warns for whitespace before the `@since` version, hence removing this test case.
    
This is because leading whitespace is now trimmed from [`tag.description`](https://github.com/google/site-kit-wp/blob/2f0e55264a54d95b431619d6314ef43ecfc2b08a/packages/eslint-plugin/rules/jsdoc-requires-since.js#L112).
    
It's possible to restore this functionality by parsing `sinceTags[ 0 ].source`, we could potentially address this in a followup issue.